### PR TITLE
Fix depracated FloatRangeField problem

### PR DESCRIPTION
**Pull request by kadircancetin**
This commit about issue #133 and incomplete pr #132 . And very thanks to @cobang , he mentored me on this pr.

### CODE REVIEW

File: drf_extra_fields/compat.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+try:
+    from django.contrib.postgres.fields import FloatRangeField
+except ImportError:
+    FloatRangeField = None
+
+try:
+    from django.contrib.postgres.fields import DecimalRangeField
+except ImportError:
+    DecimalRangeField = None
```
Comment (cobang) [inline drf_extra_fields/compat.py:9]: Do we need to check the compatibility of `DecimalRangeField`? Looks like it is added in Django2.2 and this package supports Django2.2 and newer versions.

Comment (alicertel) [reply drf_extra_fields/compat.py:9]: @cobang You are right.

File: tests/test_fields.py [hunk 2]
```diff
@@ -492,6 +504,84 @@ def test_no_source_on_child(self):
...
+        (NumericRange(**{'lower': '1.1', 'upper': '2'}),
+         {'lower': '1.1', 'upper': '2', 'bounds': '[)'}),
+        (NumericRange(**{'empty': True}), {'empty': True}),
+        (NumericRange(), {'bounds': '[)', 'lower': None, 'upper': None}),
+    ]
+    field = DecimalRangeField()
+
+    def test_no_source_on_child(self):
+        with pytest.raises(AssertionError) as exc_info:
+            IntegerRangeField(child=serializers.IntegerField(source='other'))
```
Comment (alicertel) [inline tests/test_fields.py:537]: `DecimalRangeField(child=serializers.DecimalField(source='other'))`?

File: tests/test_fields.py [hunk 3]
```diff
@@ -492,6 +504,84 @@ def test_no_source_on_child(self):
...
+    ]
+
+    def test_no_source_on_child(self):
+        with pytest.raises(AssertionError) as exc_info:
+            IntegerRangeField(child=serializers.IntegerField(source='other'))
+
+        assert str(exc_info.value) == (
+            "The `source` argument is not meaningful when applied to a `child=` field. "
+            "Remove `source=` from the field declaration."
+        )
```
Comment (alicertel) [inline tests/test_fields.py:582]: We can remove test_no_source_on_child since it's tested in `TestDecimalRangeField`

File: tests/test_fields.py [hunk 4]
```diff
@@ -492,6 +504,84 @@ def test_no_source_on_child(self):
...
+
+
 class TestFloatRangeField(FieldValues):
```
Comment (alicertel) [inline tests/test_fields.py:585]: Shouldn't  `TestFloatRangeField` be run if `compat.FloatRangeField` is not None?
Earlier there was a such a line which is removed recently.
```
@pytest.mark.skipif(django.VERSION < (1, 8) or compat.postgres_fields is None,
                    reason='RangeField is only available for django1.8+'
                    ' and with psycopg2.')
```

I think we need such a decorator to skip test if Floatrange is not supported.

File: tests/test_model_serializer.py [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+    integer_range_field = IntegerRangeField()
+
+    class Meta:
+        app_label = 'tests'
+
+
+if compat.FloatRangeField:
+    class TestFloatRangeFieldMapping(TestCase):
+
+        def test_decimal_range_field(self):
```
Comment (alicertel) [inline tests/test_model_serializer.py:31]: test_float_range_field?

**Pull request merged**